pkg/instrumentation/sdk_test.go

```diff
@@ -865,7 +865,7 @@ func TestInjectDotNet(t *testing.T) {
 				{
 					Name:    dotnetInitContainerName,
 					Image:   "img:1",
-					Command: []string{"cp", "-a", "/autoinstrumentation/.", dotnetInstrMountPath},
+					Command: []string{"cp", "-r", "/autoinstrumentation/.", dotnetInstrMountPath},
```

Comment:
is cp -r available on the distributions we use? I tested it on my mac and it failed

Reply:
I think we use busybox everywhere, so it should work.

Reply:
yeah i think just some extra confirmation that this worked on a local kind cluster would be good

Reply:
The current E2E are testing this if I'm not wrong @IshwarKanse

Reply:
So the current e2e tests are not checking the status of the containers in the pod. I have fixed that with the PR https://github.com/open-telemetry/opentelemetry-operator/pull/2702 with the changes, we check all the containers in the instrumented app pod are running and the test will fail if any of the container fails.